operator/runtime/stagenet/src/configs/runtime_params.rs

```diff
@@ -350,10 +350,21 @@ pub mod dynamic_params {
 
         #[codec(index = 37)]
         #[allow(non_upper_case_globals)]
-        /// Targeted annual inflation rate.
-        /// Default: 5% per annum
-        /// This rate is divided across all eras in a year to calculate per-era inflation.
-        pub static InflationTargetedAnnualRate: Perbill = Perbill::from_percent(5);
+        /// Fixed annual inflation amount in base units (wei).
+        ///
+        /// This implements **linear (non-compounding) inflation** where a fixed amount of tokens
+        /// is minted annually, regardless of current total supply. This ensures:
+        /// - Consistent, predictable rewards for validators and stakers
+        /// - Publicly auditable emissions on the blockchain
+        /// - 5% of genesis supply, not 5% of current supply
+        ///
+        /// Formula: 5_000_000 * HAVE * SUPPLY_FACTOR
+        /// - Base: 5M HAVE annual inflation (5% of 100M base supply)
+        /// - Stagenet (SUPPLY_FACTOR=1): 5M HAVE annual (5% of 100M)
+        ///
+        /// The annual amount is divided equally across all eras in a year (~1461 eras with 6-hour eras).
+        /// Per-era inflation ≈ 3,422 HAVE (stagenet)
+        pub static InflationAnnualAmount: Balance = 5_000_000 * HAVE * SUPPLY_FACTOR;
```

Comment:
We need to be careful with reusing indexes and changing types. This index `37` was a `Perbill` before, and I'm not sure, but if this is live on testnet, and was changed as a parameter, we might be miss reading the old value. Probably unimportant now, but a warning for the future